data-otservbr-global/npc/lokur.lua

```diff
@@ -95,6 +95,33 @@ local function creatureSayCallback(npc, creature, type, message)
 	npc:parseGuildBank(message, npc, creature, playerId, npcHandler)
 	-- Normal messages
 	npc:parseBankMessages(message, npc, creature, npcHandler)
+
+	if MsgContains(message, "ticket") then
```

Comment:
Duplicating the code: https://github.com/opentibiabr/canary/blob/102b8877e4f91ac642e7cbaf6737f5c2aa50e7d6/data-otservbr-global/npc/lokur.lua#L66C2-L66C40
https://github.com/opentibiabr/canary/blob/102b8877e4f91ac642e7cbaf6737f5c2aa50e7d6/data-otservbr-global/npc/lokur.lua#L99

Reply:
Man, you are right, i'm sorry about that, dont know what happened. I'll change it